### DISCUSSION TITLE
Can I create a webview_builder without specify the initialize URL

### DISCUSSION THREAD

**Discussion by crackleeeessyp** [tauri-apps/tauri#11965] [Q&A]
Seems currently `WebviewWindowBuilder` needs to specify the URL, can I create it without URL, and latter load another URL

```rust
let mut webview_builder = tauri::WebviewWindowBuilder::new(
    app,
    "main",
    tauri::WebviewUrl::App("https://www.bing.com".into()),
)
```

I don't want to use about:blank, because it will add an history item to the WebView?

**Comment by FabianLars** [comment]
This is currently not supported but feel free to open a feature request (as a GitHub issue) for this. No promise that we can make it work though, I can't remember the technical details right now, but I feel like that at least once you call build() on the Builder you need an entry point url...

**Reply by crackleeeessyp** [reply]
Thanks very much for your information
